src/rockstor/storageadmin/static/storageadmin/js/views/shares.js

```diff
@@ -138,5 +139,16 @@ SharesView = RockstorLayoutView.extend({
 			}
 			return new Handlebars.SafeString(html);
 		});
+
+        Handlebars.registerHelper('isSystemShare', function(id, pool_id) {
+
+            //During Rockstor installation we create first pool with pool_id == 1
+            //and root and home shares with id 1 and 2; this let us filter on ids
+            //instead of share names
```

Comment:
Your reasoning is correct here, but please see my previous comment. For the OS Pool, the pool.role is already set to "root" during disk scan process. Do you have any concerns using that instead of id == 1?